k6/stress/my_page.js

```diff
@@ -7,11 +7,11 @@ export let errorRate = new Rate('errors');
 
 export let options = {
   stages: [
-    {duration: '3m', target: 100}, // Stage: ramp up
-    {duration: '3m', target: 200}, // Stage: ramp up
-    {duration: '3m', target: 300}, // Stage: ramp up
-    {duration: '3m', target: 400}, // Stage: ramp up
-    {duration: '3m', target: 500}, // Stage: ramp up
+    {duration: '2m', target: 40}, // Stage: ramp up
+    {duration: '2m', target: 80}, // Stage: ramp up
+    {duration: '2m', target: 120}, // Stage: ramp up
+    {duration: '2m', target: 160}, // Stage: ramp up
+    {duration: '2m', target: 200}, // Stage: ramp up
```

Comment:
```suggestion
    {duration: '10s', target: 40},
    {duration: '2m', target: 40}, // Stage: ramp up
    {duration: '10s', target: 80},    
    {duration: '2m', target: 80}, // Stage: ramp up
    {duration: '10s', target: 120},     
    {duration: '2m', target: 120}, // Stage: ramp up
...
```

특정 부하에서의 상태를 확인하려면 증가하는 구간과 유지 구간을 분리해야 해요.
그렇지 않으면 duration 기간동안 target의 부하까지 점진적으로 부하를 증가시켜나가요

Reply:
네 말씀 감사합니다. 앞으로 중간에 유지 구간을 두어서 테스트를 진행해보겠습니다. 👍